pkg/jobrunaggregator/jobrunaggregatorlib/disruption.go

```diff
@@ -48,6 +49,44 @@ type BackendDisruption struct {
 	DisruptionMessages []string
 }
 
+// defined in origin/platformidentification/types.go
```

Comment:
just a nit but godoc convention would be to have the name of the thing at the start of the comment,  "ClusterData defined in..."

https://go.dev/blog/godoc